### ISSUE TITLE
Support bulk deletion of stale identities

### ISSUE THREAD

**Issue by js-sketch** [Flagsmith/flagsmith#291]
As time passes a project/environment will accumulate identities. A lot of these identities may have gone unused for a long time and is no longer relevant/can be considered unused. It would be helpful if these could be deleted.

I've not looked too much into the source code, but this would at least require:

1. Adding a new API end point for deleting identities in bulk (payload being a JSON array?)
2. Adding a new 'lastSeen' timestamp or similar field to the identity model and presenting this info in the API

A current workaround is to use traits to upload the timestamp.

**Comment by dabeeeenster** [comment]
Yes - this is definitely something we have discussed internally and will be looking to implement. 

A better UX might be to allow Organisation/Project administrators to define an optional `maxAge` parameter for Identities within an Environment. Then run a nightly job that removes any identities that have a `lastSeen` timestamp older than the `maxAge` setting.
